### PR TITLE
Add HTRflow as a library

### PR DESCRIPTION
**Pull request by Borg93**
This PR adds https://github.com/AI-Riksarkivet/htrflow/ as a library.

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries.ts:320]: Hi @Borg93 - Thanks for the PR, I don't see any models with tags `htrflow`. Once this PR is in-effect it on model repos with the aforementioned tag (`htrflow`).

File: packages/tasks/src/model-libraries-snippets.ts
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries-snippets.ts:367]: We push to make the model snippets more complete, it might make sense to have a more complete python snippet like here: https://ai-riksarkivet.github.io/htrflow/latest/getting_started/pipeline.html#nested-segmentation

Based on the model checkpoints you could customise this snippet per task.

**Review by Vaibhavs10 — COMMENTED** [review]
Thanks for the PR, I left some comments to make it a bit more impactful.

File: packages/tasks/src/model-libraries.ts
Comment (Borg93) [reply packages/tasks/src/model-libraries.ts:320]: Sure! Added some now : https://huggingface.co/models?other=htrflow

File: packages/tasks/src/model-libraries-snippets.ts
Comment (Borg93) [reply packages/tasks/src/model-libraries-snippets.ts:367]: Hi, I'm just how we should document it, since we are actually trying to import from different framework under the hood.. see snippet below as an example:

```py
from htrflow.pipeline.steps import Segmentation
from htrflow.pipeline.pipeline import Pipeline
from htrflow.volume.volume import Collection
from htrflow.models.ultralytics.yolo import YOLO

collection = Collection.from_directory("./")

# Create additional steps
segmentation_step = Segmentation(
    model_class=YOLO,
    model_kwargs={'model': 'Riksarkivet/yolov9-regions-1'},
    generation_kwargs={}
)

# Create the pipeline with multiple steps
pipeline = Pipeline(
    [
        segmentation_step,
 
    ]
)

pipeline.run(collection)
```

Comment (Borg93) [reply packages/tasks/src/model-libraries-snippets.ts:367]: Do we need to unify this or what do you think?

Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries-snippets.ts:367]: The snippet in the latest commit looks good to me! 👍

File: packages/tasks/src/model-libraries-snippets.ts
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries-snippets.ts:374]:
```suggestion
from htrflow.pipeline.steps import Task
```
I think the quickstart documentation is enough for people to understand how to go about it.

File: packages/tasks/src/model-libraries-snippets.ts
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries-snippets.ts:375]:
```suggestion
from htrflow.models.framework.model import ModelClass
```
Same as above

File: packages/tasks/src/model-libraries-snippets.ts
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries-snippets.ts:368]:
```suggestion
	`# HTRflow can be used through CLI or Python
```

small nit

**Review by Vaibhavs10 — APPROVED** [review]
I think this is in a good shape. Let's wait for another review before merging. Thanks again for opening the PR! 🤗

File: packages/tasks/src/model-libraries-snippets.ts
Comment (pcuenca) [inline packages/tasks/src/model-libraries-snippets.ts:371]:
```suggestion
        , # Python usage
```

Comment (Wauplin) [inline packages/tasks/src/model-libraries-snippets.ts:371]:
```suggestion
# see docs: https://ai-riksarkivet.github.io/htrflow/latest/getting_started/quick_start.html
htrflow pipeline <path/to/pipeline.yaml> <path/to/image>`,
        `# Python usage
```

I think this is what's needed to keep 2 snippets while fixing the lint CI

Comment (pcuenca) [reply packages/tasks/src/model-libraries-snippets.ts:371]: That's better than my suggestion!

**Review by Wauplin — APPROVED** [review]
All good now! CI is green so time to merge :)

**Comment by Borg93** [comment]
@Wauplin  btw, when can we expect to see the integration on hf.co? :)

**Pull request merged**
